packages/components/src/InputPhoneNumber/InputPhoneNumber.stories.mdx

```diff
@@ -0,0 +1,78 @@
+import { useState } from "react";
+import { ArgsTable, Canvas, Meta, Story } from "@storybook/addon-docs";
+import { ExampleWithHooks } from "mdxUtils/ExampleWithHooks";
+import { InputPhoneNumber } from "@jobber/components/InputPhoneNumber";
+
+<Meta title="Components/InputPhoneNumber" component={InputPhoneNumber} />
+
+# InputPhoneNumber
+
+Automatically format your user's phone number as they type.
+
```

Comment:
Maybe we should add a warning about the current scope/design/assumptions about country code... that we're not including handling of it for now... and that we're assuming North American phone formats only...

Reply:
Tru tru. Should it be up top or here?
![image](https://user-images.githubusercontent.com/15986172/235220836-da5779be-4b30-4450-93e1-4e01921719b6.png)
https://feat-input-phone-number.atlantis.pages.dev/?path=/docs/components-inputphonenumber--input-phone-number#with-country-code-as-a-prefix

Reply:
I think up top, for anyone "first discovering we have this" / "considering this for use in their feature".

Reply:
> and that we're assuming North American phone formats only...

I need to confirm this from someone who is actually not in NA. iOS and Android seems to not care about the format and both chose `(***) ***-****` as _the_ format for everyone. Changing your phone's region, which changes a lot of your locale settings, somehow keeps the phone format the same. So this might just be a classic, if it's good enough for Apple and Google, it's good enough for us!

Reply:
Interesting! That would be excellent (and not entirely surprising) -- it was pre-internet when I was travelling in Europe and maybe things have standardized since then!